src/query/query_aggregate.hpp

```diff
@@ -29,7 +29,8 @@
 
 #include "external_sort.h"    // SORTKEY_INFO
 #include "query_list.h"
-#include "storage_common.h"   // AGGREGATE_HASH_STATE, SCAN_CODE, FUNC_TYPE
+#include "storage_common.h"   // AGGREGATE_HASH_STATE, SCAN_CODE
```

Comment:
The comment looks not fit to explain the enum's.

Reply:
AGGREGATE_HASH_STATE and SCAN_CODE are declared in storage_common.h